base/reducedim.jl

```diff
@@ -277,6 +277,158 @@ reducedim(op, A::AbstractArray, region) = mapreducedim(identity, op, A, region)
 
 
 ##### Specific reduction functions #####
+"""
+    sum(A, dims)
+
+Sum elements of an array over the given dimensions.
+
+```jldoctest
+julia> A = [1 2; 3 4]
+2×2 Array{Int64,2}:
+ 1  2
+ 3  4
+
+julia> sum(A, 1)
+1×2 Array{Int64,2}:
+ 4  6
+
+julia> sum(A, 2)
+2×1 Array{Int64,2}:
+ 3
+ 7
+```
+"""
+function sum(A, dims) end
```

Comment:
Thank you for the feedback on both counts! I'll try that. I realized some of my updates were breaking things but I hadn't figured out what I was doing wrong yet.